packages/common/pipes/parse-uuid.pipe.ts

```diff
@@ -45,4 +51,12 @@ export class ParseUUIDPipe implements PipeTransform<string> {
     }
     return value;
   }
+
+  protected isUUID(str: unknown, version = 'all') {
+    if (typeof str !== 'string') {
```

Comment:
let's DRY this

```suggestion
    if (!isString(str)) {
```

`isString` is imported from `../utils/shared.utils`

Reply:
Fixed, thanks